.github/workflows/rolling-semi-binary-build.yml

```diff
@@ -12,6 +12,11 @@ on:
     # Run every morning to detect flakiness and broken dependencies
     - cron: '33 1 * * *'
 
+#TODO(christophfroehlich): remove once it is released
+# otherwise ros-gz won't build
+env:
+  GZ_VERSION: fortress
```

Comment:
This does not work, the environment variable seems to be not available while while running rosdep. Is there a way to pass it?

Reply:
When running `rosdep`, you can use `--skip-keys` and you can install manually the Gazebo version. Maybe this will help as an inspiration https://github.com/ros-controls/gz_ros2_control/blob/master/.github/workflows/ci.yaml

Reply:
thanks for pointing in this direction, I'll have a look

Reply:
I'd need to change the CI files, because we use the ros-industrial CI here. I hope that everything gets released at some point..